mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -72,52 +75,41 @@ public void init() throws DBException {
 
             // initialize MongoDb driver
             Properties props = getProperties();
+
+            // Just use the standard connection format URL
+            // http://docs.mongodb.org/manual/reference/connection-string/
             String url = props.getProperty("mongodb.url",
-                    "mongodb://localhost:27017");
-            database = props.getProperty("mongodb.database", "ycsb");
-            String writeConcernType = props.getProperty("mongodb.writeConcern",
```

Comment:
since we're moving the write concern from a property to a connection URL, these changes will have to go into a new major version of YCSB since people with existing test rigs for running MongoDB will break.

That's fine, just a heads up about timing.